prov/gni/include/fi_ext_gni.h

```diff
@@ -57,6 +57,7 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
 			   GNI_MAX_RETRANSMITS,
 			   GNI_ERR_INJECT_COUNT,
 			   GNI_MR_CACHE_LAZY_DEREG,
+			   GNI_MR_CACHE_UDREG,
```

Comment:
I think  we could open a issue for making no-caching a thing.


--Jim

________________________________
From: Sung-Eun Choi <notifications@github.com>
Sent: Wednesday, May 25, 2016 8:40:55 AM
To: ofi-cray/libfabric-cray
Cc: James Swaro; Author
Subject: Re: [ofi-cray/libfabric-cray] Add udreg option for memory registration (#826)


In prov/gni/include/fi_ext_gni.h<https://github.com/ofi-cray/libfabric-cray/pull/826#discussion_r64573811>:

> @@ -57,6 +57,7 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
>                          GNI_MAX_RETRANSMITS,
>                          GNI_ERR_INJECT_COUNT,
>                          GNI_MR_CACHE_LAZY_DEREG,
> +                        GNI_MR_CACHE_UDREG,


Do we ever think people might want the option for no caching? Say for example a client chooses to implement their own cache they might not want our cache. If so, maybe this should be GNI_MR_CACHE_TYPE (or something like that) and we could parse strings like "default", "none", "udreg", etc.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ofi-cray/libfabric-cray/pull/826/files/11507947c924fbe9ffc39c38fe0c233ce3979b21#r64573811>